cypress/support/fragments/topMenu.js

```diff
@@ -24,5 +24,6 @@ export default {
   checkInPath: '/checkin',
   checkOutPath: '/checkout',
   receivingPath: '/receiving',
-  bulkEditPath: '/bulk-edit'
+  bulkEditPath: '/bulk-edit',
+  mappingProfilesPath: '/settings/data-import/mapping-profiles'
```

Comment:
we have already this path in the settingsMenu fragment. It is defined by the variable mappingProfilePath

Reply:
Done